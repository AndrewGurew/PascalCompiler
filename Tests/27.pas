function MaxNumber(a,b: integer): integer;
   var max: integer;
begin
   if a>b then max:=a else max:=b;
   MaxNumber := max;
end;