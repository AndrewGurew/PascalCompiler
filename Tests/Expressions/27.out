Unknown identifier a
Unknown identifier b
Unknown identifier a
⎬-
 ⎬2